commonskmm/src/commonMain/kotlin/org/dhis2/mobile/commons/extensions/StringExtensions.kt

```
@@ -69,9 +69,22 @@ suspend fun String.userFriendlyValue(
     }
 }
 
-fun String.toDateTimeFormat() = LocalDateTime.parse(this).format(dateTimeFormat)
-fun String.toDateFormat() = LocalDate.parse(this).format(dateFormat)
-fun String.toTimeFormat() = LocalTime.parse(this).format(timeFormat)
+fun String.toDateTimeFormat() = try {
+    LocalDateTime.parse(this).format(dateTimeFormat)
+} catch (e: Exception) {
```

I think that this might hide potential errors, what do you think about sending the trace to sentry as a handled error to see if it happend and how can this be configured?

---

good idea!

---

Marking this as resolved.